src/komodo_curve25519.h

```diff
@@ -81,7 +82,7 @@ bits320 fexpand(bits256 basepoint)
     return(out);
 }
 
-#if __amd64__
```

Comment:
These should be checking for any 64 bit platform, not only amd64, which doesn't include ARMv8

Reply:
This is a bug in upstream KMD and/or the library it pulled that code from

Reply:
`#if defined(__amd64__) || defined(__aarch64__)` fixes this for our current needs, but still does not detect any 64 bit platform. We don't support ppc64 for other reasons, so we don't list it.